### DISCUSSION TITLE
Is it Compuslory to Explicitly Call for Cleanup?

### DISCUSSION THREAD

**Discussion by manavm1990** [vitest-dev/vitest#2203] [General]
https://github.com/vitest-dev/vitest/blob/c00bc88258566bdf2ab5b3d7f1ffd543a782f6f8/examples/react-testing-lib/src/utils/test-utils.tsx#L5

> [Cleanup](https://testing-library.com/docs/react-testing-library/api#cleanup) is called after each test automatically by default if the testing framework you're using supports the afterEach global (like mocha, Jest, and Jasmine).
> [Source](https://testing-library.com/docs/react-testing-library/setup/#skipping-auto-cleanup)

**Comment by sheremet-va** [comment]
It's not called automatically by default in Vitest
